### DISCUSSION TITLE
Extract the list of all cosmic objects

### DISCUSSION THREAD

**Discussion by SebastianCikes** [Stellarium/stellarium#3387] [General]
I'm working on a remote control API project, is it possible to somehow extract a list of the names of all cosmic objects (planets, stars, asteroids, etc) to then be used as a request to the API?
(url = f"http://localhost:8090/api/objects/info?name={selected_item}&format=json")

**Comment by gzotti** [comment]
Nothing more than our API documents on the dev website. There may be more names that could be extracted from the program itself. Consider searching for a name where the search is forwarded to Simbad. Names of minor bodies depend on the elements available in the user's ssystem_minor.ini. Likewise satellites may differ between installations. Or star names which may depend on the currently loaded skyculture.

**Reply by SebastianCikes** [reply]
Thanks, I've extracted a list of bodies from ssystem_major.ini
